src/webgpu/web_platform/external_texture/video.spec.ts

```diff
@@ -26,36 +26,42 @@ const kVideoExpectations = [
   {
     videoSource: 'red-green.webmvp8.webm',
     colorSpace: 'REC601',
```

Comment:
aside: Shouldn't `colorSpace` be a private param as well? `_colorSpace`

(this comment is obsolete if you do the other suggestion though)

Reply:
Want to keep it as parameter to map fail case to color space info easily .

Reply:
Think it again. We could easily find this info in the table so I agree to remove this from parameter and place it into the table.